app/src/main/java/org/simple/clinic/widgets/Views.kt

```diff
@@ -279,3 +279,9 @@ fun ViewPropertyAnimator.setDuration(duration: Duration): ViewPropertyAnimator {
 
 val Int.dp: Int
   get() = (Resources.getSystem().displayMetrics.density * this).toInt()
+
+inline var View.isVisible: Boolean
```

Comment:
Can we make it only a computed `val` property which doesn't have a setter? A caller might assume setting `isVisible` to `true` makes it `INVISIBLE` from the name, but the implementation sets it to `GONE`.

Let callers use the platform `setVisibility` method for changing it as necessary.

Reply:
Sure 👍